<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SaveFiles</class>
 <widget class="QDialog" name="SaveFiles">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>342</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SaveFiles</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="warnLabel">
         <property name="font">
          <font>
           <family>Sans Serif</family>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>There are x documents with unsaved changes. Save changes before closing?</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="selectLabel">
         <property name="text">
          <string>Select the documents you want to save:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableView" name="fileTable"/>
       </item>
       <item>
        <widget class="QLabel" name="lostLabel">
         <property name="text">
          <string>All unsaved changes will be permanently lost.</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="closeWithoutSaveButton">
         <property name="text">
          <string>Close &amp;without Saving</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cancelButton">
         <property name="text">
          <string>&amp;Cancel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="saveButton">
         <property name="text">
          <string>&amp;Save</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
